nextflow.config

```diff
@@ -14,9 +14,10 @@ params {
   flowcell = false
   kit = false
   barcode_kit = false
+  guppy_config = false
   guppyGPU = false
   gpu_device = 'auto'
-  gpu_cluster_options = ''
+  gpu_cluster_options = false
```

Comment:
This line sneaked in from a previous fix?

Reply:
Should be ok no? It's just set to `false` by default.

Reply:
Yeah, sorry, mostly curious! :)